community-plugins.json

```diff
@@ -5788,5 +5788,12 @@
         "author": "Ryan Gomba",
         "repo": "ryangomba/obsidian-todo-sort",
         "branch": "main"
+    },
+    {
+        "id": "obsidian-file-chucker",
+        "name": "Obsidian file chucker",
```

Comment:
We're asking plugins to avoid using the word "Obsidian" in their plugin name to avoid desaturating the brand name and reserve the naming for first-party plugins (see: Obsidian Publish, Obsidian Sync).

Reply:
Have changed this to just "file-chucker"